aes-ctr/src/lib.rs

```diff
@@ -9,8 +9,8 @@
 //! it either by using `RUSTFLAGS="-C target-feature=+aes,+ssse3"` or by editing
 //! your `.cargo/config`. (`sse2` target feature is usually enabled by default)
 //!
-//! # Warning
-//! This crate does not provide any authentification! Thus ciphertext integrity
```

Comment:
I changed all of these, mainly because "authentification" is somewhat peculiar (I'd normally call it "authentication")

Reply:
It's a common misspelling for many Europeans. :)